app/webpack/stylesheets/_shame.scss

```
@@ -206,7 +206,6 @@ form {
 // claims-table.SCSS
```

As noted in the linting tests;

```suggestion
/* claims-table.SCSS */
```